### ISSUE TITLE
Unable to dismiss as a keyboard user

### ISSUE THREAD

**Issue by Kanx** [insites/cookieconsent#249]
We use cookie consent on our websites and recently had an accessibility audit undertaken. One of the issues raised was around not being able to accept/dismiss the modal as a keyboard user (hitting return on the dismiss button).

I was able to replicate this issue on our site as well as any other site using the plug-in.

Is this intended behaviour? I've attempted a workaround by listening for a `keydown` while focusing on the `.cc-dismiss` element, but I'm unable to programatically trigger the dismiss event.

Edit: Just noticed this pull request addresses the issue https://github.com/insites/cookieconsent/pull/213

**Comment by roelvangils** [comment]
It’s my ultimate Christmas wish that someone merges that pull request ;)

**Comment by alexmorleyfinch** [comment]
I don't think adding a keypress to auto accept would be a good idea, as people could accidentally tap a key without realising they gave consent. You could make the buttons tabbable so when you press tab the consent button is highlighted, and then when pressing enter it would be just like clicking on the button
